posts/2024-07-09-pathogfair-preprint.md

```diff
@@ -3,16 +3,21 @@ media:
   - mastodon-eu-freiburg
   - matrix-eu-announce
   - linkedin-galaxyproject
```

Comment:
What is happening if we merge this? Does it post again to matrix, linkedid?

Reply:
Although it previews for all social media platforms, it should only process Bluesky.
I will fix this to show only the platforms that are being processed later.

Reply:
But don't merge it yet.
Let me make sure about it.

Reply:
I created the fixture PR to prevent the additional previews: #73

Reply:
I found a bug while testing this post. Please approve PR #74 to fix it.